scripts/ask_qdrant.py

```
@@ -0,0 +1,51 @@
+
+    ensure_env("OPENAI_API_KEY")
+    ensure_env("QDRANT_URL")
+    ensure_env("NEO4J_URI")
+
+    log = {
+        "timestamp": _dt.datetime.utcnow().isoformat() + "Z",
+        "operation": "ask_qdrant",
+        "question": args.question,
+        "top_k": int(args.top_k),
+        "status": "skipped",
+        "message": "Stub implementation – full retrieval delivered in Story 2.5",
```

Ruff (0.13.1)

String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)